### ISSUE TITLE
[request] Open download folder button

### ISSUE THREAD

**Issue by makedir** [lay295/TwitchDownloader#855]
### Checklist

- [X] I have checked the [issue page](https://github.com/lay295/TwitchDownloader/issues?q=is%3Aissue) for duplicates
- [X] I am running the latest version ([download here](https://github.com/lay295/TwitchDownloader/releases/latest))

### Write your feature request here

Could you please implement a button in the main gui which opens the (last used) folder in Explorer where the video was saved to.

**Comment by ScrubN** [comment]
This exists now (since January) for items sent to the task queue.

**Comment by Ariskrisen** [comment]
I agree, it would be great to see a button in the main interface (under the settings button) that opens the (last used) folder in Explorer where the video will be saved, because it's not convenient now

**Comment by makedir** [comment]
I have no idea how developers are so limited sometimes in having a vision for their own applications. If I were the developer of this app, I would implement little things like this non stop. This is just one example of many.
